html2asketch/base.js

```diff
@@ -1,10 +1,8 @@
-import {generateID} from './helpers/utils';
-
 class Base {
-  constructor() {
+  constructor({id} = {}) {
```

Comment:
What about `constructor({ id } = { id: generateID()})` for backwards compatibility?

Reply:
can you clarify backwards compatibility in the context of `*.asketch` artifacts and sketch plugins?

Reply:
Sure, I'm talking about backwards compatibility with tools that depend on html-sketchapp and use the Base class. The previous code would generate a randomID if Base was instantiated with no arguments to the constructor, but this changes that behavior.

Reply:
but tools dont rely on those ids, do they? so it wouldnt be a breaking change, right?

Reply:
I don't think we can assume that no one depends on those. +1 for @elliotdickison suggestion